### PR TITLE
docs: Resolve mismatch between latest and 1.7.0

### LINKED ISSUE #613: Documentation mismatch
**Describe the bug**
I would expect documentation in [documentation/docs](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/tree/main/documentation/docs) being exactly the same or newer than the latest released version (Currently [documentation/versioned_docs/version-1.7.0](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/tree/main/documentation/versioned_docs/version-1.7.0) ). However, this is not the case. One example:

Last change Feb 14: [documentation/docs/configuration/delete_config.md](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/b73404dd69fc9e8b37b8bd1e38b7edda32312c8e/documentation/docs/configuration/delete_config.md)

Last change Feb 22: [documentation/versioned_docs/version-1.7.0/configuration/delete_config.md](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/b73404dd69fc9e8b37b8bd1e38b7edda32312c8e/documentation/versioned_docs/version-1.7.0/configuration/delete_config.md)

### PR DESCRIPTION
**Pull request by tobigremmer-dt**
Resolves #613

### CODE REVIEW

File: documentation/docs/commands/deploying-projects.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ sidebar_position: 2
 
 # Deploy projects
 
-The tool allows for deploying a configuration or a set of configurations in the form of project(s). A project is a folder containing files that define configurations to be deployed to a environment or a group of environments. This is done by passing the `--project` flag (or `-p` for short).
+The `monaco` tool can deploy a configuration or a set of configurations in the form of project(s). A project is a folder containing files that define the configurations to be deployed to a environment or a group of environments. This is done by passing the `--project` flag (or `-p` for short).
```
Comment (Giulia-dipietro) [inline documentation/docs/commands/deploying-projects.md:7]: Can you please change 'monaco' to Monaco, so that it's consistent.

Comment (tobigremmer-dt) [reply documentation/docs/commands/deploying-projects.md]: Done

File: documentation/docs/commands/deploying-projects.md [hunk 2]
```diff
@@ -23,16 +23,16 @@ Below you find a few samples on how to run the tool to deploy your configuration
 If `project` contains additional sub-projects, then all projects are deployed recursively. If `project` depends on different projects under the same root,
 those are also deployed.
 
-Multiple projects could be specified by `-p="projectA, projectB, projectC/subproject"`.
+Multiple projects can be specified by `-p="projectA, projectB, projectC/subproject"`.
 
-To deploy configuration the tool will need a valid API Token(s) for the given environments defined as `environment variables` - you can define the name of that env var in the environments file.
+To deploy the configuration, `monaco` needs a valid API Token(s) for the each environment.  These are defined as `environment variables`; you can define the name of that env var in the environments file that is specified as an argument to the `-e` option.
```
Comment (Giulia-dipietro) [inline documentation/docs/commands/deploying-projects.md:28]: Can you please change 'monaco' to Monaco, so that it's consistent.

**Review by Giulia-dipietro — CHANGES_REQUESTED** [review]
Hi, thanks for your changes. 
Can you please make those small edits in the comments before I approve it?

**Comment by tobigremmer-dt** [comment]
All updates have been made

**Review by Giulia-dipietro — APPROVED** [review]

**Pull request merged**